### ISSUE TITLE
viewWillAppear doesn't work when back button click from pushed viewcontroller

### ISSUE THREAD

**Issue by chengte99** [PageMenu/PageMenu#410]
I add a tableview controller in pagemenu. The tableview controller will list data, and once tap selected data will push the detail viewcontroller for display detail info. The problem is the tableview controller viewWillAppear doesn't work when I tap navigation back botton from detail viewcontroller.
I want to update the tableview data list if I did some change in detail viewcontroller and back to tableview, in order to this so I need viewWillAppear working. Please help, or have any another way can solve my question? Thanks.

**Comment by chetansrm** [comment]
Yeah! same issue I faced.So the way I selected to solve this problem is like below.
Suppose you are having a containerVC where you added tableviewController in pageMenu ,save containerVC object in tableViewController.When you pressed back button from detailsVC.ViewWillAppar() will be called of tableViewVC at the same time call viewWIllAppear() of containerVC with the saved object previously.Make sure that you have added tableViewVC in pageMenu in viewWillAppear() of containerVC.
